src/EditorFeatures/Test/BraceHighlighting/AbstractBraceHighlightingTests.cs

```diff
@@ -19,6 +20,8 @@ public abstract class AbstractBraceHighlightingTests
         {
             using (var workspace = await CreateWorkspaceAsync(markup))
             {
+                WpfTestCase.RequireWpfFact($"{nameof(AbstractBraceHighlightingTests)}.{nameof(TestBraceHighlightingAsync)} creates asynchronous taggers");
```

Comment:
Consider adding [CallerMemberInfo] to RequireWpfFact.  Would save you this type of typing.